frontend-react/yarn.lock

```diff
@@ -1857,30 +1855,33 @@
     webcrypto-shim "^0.1.5"
     xhr2 "0.1.3"
 
-"@okta/okta-react@^6.4.1":
-  version "6.5.0"
-  resolved "https://registry.yarnpkg.com/@okta/okta-react/-/okta-react-6.5.0.tgz#bf22561a57c631717daa201b6f323823ba0bc08a"
-  integrity sha512-hDt4dswFIPp/JVyrbgrwxM3J9QzqJrd5ml28FKh2BvPZSqowRnAg7xdbMiaP6omLbLiygbnnKvkdVrrtB+5n0g==
+"@okta/okta-react@^6.7.0":
+  version "6.7.0"
+  resolved "https://registry.yarnpkg.com/@okta/okta-react/-/okta-react-6.7.0.tgz#6d57de781b163cbe8ed31a3bece4fed136d37b54"
+  integrity sha512-jIx1dsPBsxYBqVcfW08dc/U1mPWG1Nswj6puRzykTBBIp3gEqLps/4dOuLx4aoWkMPkwx+fF1eXfzJvhq87NcA==
   dependencies:
     "@babel/runtime" "^7.11.2"
     compare-versions "^4.1.2"
 
-"@okta/okta-signin-widget@^6.0.1":
-  version "6.4.3"
-  resolved "https://registry.yarnpkg.com/@okta/okta-signin-widget/-/okta-signin-widget-6.4.3.tgz#2108a63eaeb7ec2b22880e36081c71c9707f6de0"
-  integrity sha512-bMx+OCskWk+Pz6fxxoMhcZleHqX+DrSzS2CNkwEEEwRYV0amfe12PI99kQ4s+VmXGa0Ctdwy6g5I6fPBKuDZKw==
+"@okta/okta-signin-widget@^7.2.1":
+  version "7.2.1"
+  resolved "https://registry.yarnpkg.com/@okta/okta-signin-widget/-/okta-signin-widget-7.2.1.tgz#407abe3ef0189a173e5f1b6155b97df9905714bb"
+  integrity sha512-MvPlnmtRSjIKhvR6Aq6EKN3k4MkaNJzQwaUYHtlSVSD6vqzqBoNDJhXg6aLKvqeLDEZZCSSBYkedIWlF4I2Ofg==
   dependencies:
-    "@okta/okta-auth-js" "6.5.1"
+    "@okta/okta-auth-js" "~7.0.0"
     "@sindresorhus/to-milliseconds" "^1.0.0"
     "@types/backbone" "^1.4.15"
     "@types/jquery" "^3.5.14"
     "@types/jqueryui" "^1.12.16"
     "@types/q" "^1.5.5"
     "@types/selectize" "^0.12.35"
     "@types/underscore" "^1.11.4"
+    chokidar "^3.5.1"
     clipboard "^1.5.16"
     cross-fetch "^3.1.5"
+    ejs "^3.1.7"
     handlebars "^4.7.7"
+    jquery "^3.6.0"
```

Comment:
Blech, I'm not a fan of this, but it's a necessary hit for the signin widget.  Looks like there's [an open issue](https://github.com/okta/okta-signin-widget/issues/512) to remove it, but that was from almost three years ago 🤔 

Obviously not a blocker for this since we're just upgrading to get off the EOL version.  Something we can keep an eye on for future enhancements when we have more tooling around front-end performance.